node/task.ts

```diff
@@ -765,15 +837,55 @@ export function cp(source: string, dest: string, options?: string, continueOnErr
  * @param     options    string -f or -n for force and no clobber 
  * @param     continueOnError optional. whether to continue on error
  */
-export function mv(source: string, dest: string, options?: string, continueOnError?: boolean): void {
-    if (options) {
-        shell.mv(options, source, dest);
+export function mv(source: string, dest: string, force: boolean = false, continueOnError?: boolean): void {
+    try {
+        if (!fs.existsSync(source)) {
+            throw new Error(loc('LIB_OperationFailed', source + ' doesnt exist'));
+        }
+        if (getPlatform() == Platform.Windows) {
+            if (fs.statSync(source).isDirectory()) {
+                mkdirP(dest);
```

Comment:
What if it already exists?

Reply:
That's fine, mkdirP creates the folder/all parents only if it/they don't exist